ui/src/app/applications/components/application-status-panel/application-status-panel.tsx

```diff
@@ -107,13 +102,14 @@ export const ApplicationStatusPanel = ({application, showDiff, showOperation, sh
                     <div className='application-status-panel__item-name' style={{marginBottom: '0.5em'}}>
                         {application.spec.syncPolicy?.automated ? 'Auto sync is enabled.' : 'Auto sync is not enabled.'}
                     </div>
-                    {application.status && application.status.sync && application.status.sync.revision && !application.spec.source.chart && (
+                    {application.status && application.status.sync && application.status.sync.revision && revision && !application.spec.source.chart && (
```

Comment:
Could `application.status.sync.revision && revision` be simplified to just `revision`?